ReactNative/ReactTwitter/core/views/main-navigator.js

```diff
@@ -100,31 +91,33 @@ var MainNavigator = React.createClass({
       <Navigator
         style={styles.appContainer}
         navigationBar={navBar}
-        initialRoute={{id: splashScreenID}}
+        initialRoute={{id: SplashScreenView.navigatorID()}}
         renderScene={this.navigatorRenderScene}
       />)
   },
 
   navigatorRenderScene (route: any, navigator: Navigator) {
     switch (route.id) {
-      case splashScreenID:
```

Comment:
I'd actually rewrite this into an object with key-values matching the case and string you need to return, so it's simpler to read and accessing the right template is just a matter of accessing a dictionary value.